frontend/src/components/pages/Default.tsx

```diff
@@ -38,17 +38,26 @@ const TeamInfoDisplay = () => {
   );
 };
 
+// Example: how to apply custom colors
 const DesignLibraryExamples = () => {
   return (
-    <div>
-      {/* Example: how to apply custom colors */}
+    <VStack spacing={4} bg="yellow.50" padding={2}>
       <ButtonGroup gap='4'>
-          <ChakraButton bg="blue.50" color="white"> Very Light Blue Button </ChakraButton>
```

Comment:
i replaced the original buttons because i think we will probably only use button variants moving forward? (rather than directly styling a button) but lmk if you still want to keep it as an example

Reply:
makes sense to me!